charts/kubewarden-controller/templates/deployment.yaml

```diff
@@ -28,7 +28,6 @@ spec:
         args:
         - --leader-elect
         - --deployments-namespace={{ .Release.Namespace }}
-        - --deployments-service-account-name={{ .Values.policyServer.serviceAccountName }}
```

Comment:
nitpicking: this isn't a change related with the CR. More changes like that must be done later on, when we switch to the new controller. Maybe we should leave that for another PR?

Reply:
It should be handled while fixing https://github.com/kubewarden/helm-charts/issues/22

Reply:
done